src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
no need to reset maging and padding for all selectors.
reset just for elements which need it